Views/Actions/components/SendButton.jsx

```diff
@@ -1,82 +1,152 @@
 import React, { useState } from "react";
-import { StyleSheet, View } from "react-native";
-import { Button, Modal } from "react-native-ui-kitten";
+import {
+  StyleSheet,
+  View,
+  Alert,
+  TouchableOpacity,
+  Text,
+  Modal,
+  TouchableWithoutFeedback
+} from "react-native";
+import normalize from "react-native-normalize";
 
 const sendDictionary = [
```

Comment:
when you have stuff like this move it to a separate file. maybe create a directory and treat it sort of as an asset